src/WindowsAuth/.nuget/NuGet.Config

```diff
@@ -0,0 +1,7 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
`NuGet` includes the `NuGet.exe` binary. Is this intentional? Or is there another way to bring this is an compile time?

Reply:
Yes - again following Nora's example.  I think I'm comfortable with leaving it here just to reduce the friction for folks unfamiliar with building .net stuff on Windows.